<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListWidget#chatWnd{
 outline:0px;
border:none;
 }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="all" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#all{
  background: #FFFFFF;
  background-color: #FFFFFF;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="chatWnd">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="muteBoard" native="true">
        <property name="minimumSize">
         <size>
          <width>290</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>290</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#muteBoard{
border-top:1px solid rgb(229, 229, 229);
border-bottom:1px solid rgb(229, 229, 229);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>214</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="styleSheet">
            <string notr="true">QCheckBox#checkBox{
  font-family: &quot;微软雅黑&quot;;
  font-size: 12px;
  color: #202020;
  line-height: 20px;
}
QCheckBox::indicator {
Width:12px;
Height:12px;
}

QCheckBox::indicator:unchecked {
        image: url(:/res/images/checkbox_unchecked.png);
}

QCheckBox::indicator:checked {
        image: url(:/res/images/checkbox_checked.png);
}

</string>
           </property>
           <property name="text">
            <string>全员禁言</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="inputBoard" native="true">
        <property name="minimumSize">
         <size>
          <width>290</width>
          <height>106</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>290</width>
          <height>106</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="inputWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>290</width>
             <height>56</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>56</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="inputTextEdit">
              <property name="styleSheet">
               <string notr="true">QPlainTextEdit#inputTextEdit{
  font-family: &quot;微软雅黑&quot;;
  font-size: 14px;
  line-height: 22px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="placeholderText">
               <string>说点什么吧~</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="sendWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>290</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>30</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="emojiBtn">
              <property name="minimumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#emojiBtn{
border-image: url(:/res/images/emoji.png)
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>15</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="picBtn">
              <property name="minimumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#picBtn{
border-image: url(:/res/images/figicon.png)
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>160</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="sendBtn">
              <property name="minimumSize">
               <size>
                <width>56</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>56</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#sendBtn{
  background: #4FACFF;
  border-radius: 2px;
  background-color: #4FACFF;
  font-family: &quot;微软雅黑&quot;;
  font-size: 12px;
  color: #FFFFFF;
  line-height: 20px
}</string>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
